homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -365,8 +363,8 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
     ATTR_PURIFY_VOLUME: XiaomiMiioSensorDescription(
         key=ATTR_PURIFY_VOLUME,
         name="Purify volume",
-        native_unit_of_measurement=VOLUME_CUBIC_METERS,
-        device_class=SensorDeviceClass.GAS,
+        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
+        device_class=SensorDeviceClass.VOLUME,
```

Comment:
Note for reviewers: adjusted to a less precise device class, as GAS refers to energy consumption.